applications/NurbsBrepApplication/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+message("**** configuring KratosNurbsBrepApplication ****")
+
+include_directories( ${CMAKE_SOURCE_DIR}/kratos )
+
+# generate variables with the sources
+set( KRATOS_NURBS_BREP_APPLICATION_SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/nurbs_brep_application.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/nurbs_brep_application_variables.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_python/nurbs_brep_python_application.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_utilities_to_python.cpp
+)
+
+# define library Kratos which defines the basic python interface
+pybind11_add_module(KratosNurbsBrepApplication MODULE THIN_LTO ${KRATOS_NURBS_BREP_APPLICATION_SOURCES})
+target_link_libraries(KratosNurbsBrepApplication KratosCore )
```

Comment:
I think this should link private otherwise there are warnings from pybind
Have a look at e.g. the MeshMovingApp

Reply:
👍 I used the `createApplication.py`. Maybe this should also be fixed there (and also some formatting stuff)

Reply:
Sure, on it. I will add it to #2480